### PR TITLE
Feature/vertically aligned search container

### LINKED ISSUE #4: Update styling

### PR DESCRIPTION
**Pull request by CorreyL**
## Preamble

This PR concerns right-justifying the `SearchContainer` component.

This PR fixes Issue #4

## Demo

search-container-vertically-aligned

For reference, this is what it looked like before:

search-prev-next

### CODE REVIEW

File: src/components/SearchContainer/SearchContainer.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#search-container {
+  float: right;
```
Comment (CorreyL) [inline src/components/SearchContainer/SearchContainer.css]: Let me know if you want me to take the time to implement a proper [`Flexbox` solution](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)

**Review by andreysaf — COMMENTED** [review]
Good so far, I think proper flexbox plus apply the same CSS as the new UI, same colours for buttons to show users that they can achieve the exact look and feel as our UI.

**Comment by CorreyL** [comment]
> Good so far, I think proper flexbox [...]

Addressed in 330b527

> plus apply the same CSS as the new UI, same colours for buttons to show users that they can achieve the exact look and feel as our UI.

Working on this next

**Comment by CorreyL** [comment]
> plus apply the same CSS as the new UI, same colours for buttons to show users that they can achieve the exact look and feel as our UI.

Implementing in [feature/search-results-list](https://github.com/PDFTron/webviewer-custom-ui/commits/feature/search-results-list) to minimize merge conflicts.

As of 330b527

latest-build

File: src/App.js [hunk 2]
```diff
@@ -136,16 +96,16 @@ const App = () => {
...
-      <SearchContainer
-        Annotations={Annotations}
-        annotManager={annotManager}
-        docViewer={docViewer}
-        searchTermRef={searchTerm}
-        searchContainerRef={searchContainerRef}
-        open={searchContainerOpen}
-      />
-      <div id="scroll-view" ref={scrollView}>
+      <div class="flexbox-container" id="scroll-view" ref={scrollView}>
```
Comment (andreysaf) [inline src/App.js:99]: class -> className

**Review by andreysaf — APPROVED** [review]
Few things that can be resolved in future PRs:

- When clicking search, auto focus on search input
- The document scrolls by clicking next with the search container

File: src/App.js (hunk 2 above)
Comment (CorreyL) [reply src/App.js:99]: Haha good eye, already fixed in my new branch in https://github.com/PDFTron/webviewer-custom-ui/commit/7b8b8afde985d504160469040ab32d1652ee6bcc 👍

**Pull request merged**